tests/test_template_init.py

```diff
@@ -51,7 +51,7 @@ def dev_platform(request: pytest.FixtureRequest) -> str:
 
 @pytest.fixture(
     scope="module",
-    params=["mkdocs", "sphinx", "I don't need documentation for my project."],
+    params=["mkdocs", "sphinx", "no"],
```

Comment:
I was fighting with this again trying to understand what part the template actually used vs shows the user.  I tried No, None, etc and it kept breaking things.

Reply:
probably a combination of `copier` being weird and `yaml` being cursed.

in yaml, `no`/`No` (without quotes) is a boolean false, and when an option's value is `None` (e.g. `some_key: `) copier treats the key as the value.